website/docs/docs/collaborate/environments/dbt-core-environments.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "dbt Core Environments"
+id: "dbt-core-environments"
+---
+
+## How do I maintain different environments with dbt Core?
```

Comment:
```suggestion
```

Reply:
sugg:
- as a best practice, avoid a title following another title with no content in between. it's a signal that there might be page IA issues. goal is to make every word count because people like to scan (versus read)

Reply:
removed title per your and @runleonarun 's feedback!